Libraries/Core/Devtools/symbolicateStackTrace.js

```diff
@@ -18,27 +18,38 @@ const {fetch} = require('fetch');
 
 import type {StackFrame} from 'parseErrorStack';
 
+function isSourcedFromDisk(sourcePath: string): boolean {
+  return !/^http/.test(sourcePath) && /[\\/]/.test(sourcePath);
+}
+
 async function symbolicateStackTrace(stack: Array<StackFrame>): Promise<Array<StackFrame>> {
   const devServer = getDevServer();
   if (!devServer.bundleLoadedFromServer) {
     throw new Error('Bundle was not loaded from the packager');
   }
-  if (SourceCode.scriptURL) {
-    for (let i = 0; i < stack.length; ++i) {
+
+  let stackCopy = stack;
```

Comment:
The thing is that `stack = [...stack]` produces a shallow copy, i.e. members of `stack` after the assignment would still remain frozen, so the old code still would throw the exception. Also modifying input parameter doesn't sound like a safe operation, so i'd still prefer to have a separate variable for any modifications, unless the modification is intentional.

Though I agree that `stackCopy` isn't a perfect name and might be confusing. Maybe use separate variable for modified stack and then do `JSON.stringify({stack: (stackCopy || stack)})`?